hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyCryptoTestsSuite.java

```diff
@@ -773,7 +774,7 @@ private HapiSpec hollowAccountCreationChargesExpectedFees() {
     private HapiSpec hollowAccountCompletionWithEthereumTransaction() {
         final Map<String, String> startingProps = new HashMap<>();
         final String CONTRACT = "Fuse";
-        return defaultHapiSpec("HollowAccountCompletionWithEthereumTransaction")
+        return onlyDefaultHapiSpec("HollowAccountCompletionWithEthereumTransaction")
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** ‏Let's not forget to revert this before merging otherwise we'll be running only this spec in this suite.

Reply:
Reverted!